ansible/roles/commcarehq/templates/localsettings.salt.py.j2

```diff
@@ -39,8 +39,8 @@ EMAIL_SMTP_HOST = "{{ localsettings.EMAIL_SMTP_HOST }}"
 EMAIL_SMTP_PORT = {{ localsettings.EMAIL_SMTP_PORT }}
 
 SERVER_EMAIL = 'commcarehq-noreply+{{ localsettings.DEPLOY_MACHINE_NAME }}@dimagi.com' #the physical server emailing - differentiate if needed
-EMAIL_SUBJECT_PREFIX = '[{{ deploy_env }}] '
-SERVER_ENVIRONMENT = '{{ deploy_env }}'
+EMAIL_SUBJECT_PREFIX = '[{{ item.deploy_env_name }}] '
+SERVER_ENVIRONMENT = '{{ item.deploy_env_name }}'
```

Comment:
I think for the time being we need to make this change to [sync-localsettings.sh](https://github.com/dimagi/commcarehq-ansible/blob/bc92f21107a8417d397424cc023fbeab7267208d/scripts/sync-localsettings.sh)